skypy/galaxy/_spectrum_loaders.py

```diff
@@ -94,37 +101,46 @@ def skypy_data_loader(module, name, *tags):
     return spectra
 
 
-def decam_loader(*bands):
-    '''load DECam bandpass filters'''
+def speclite_loader(name, *bands):
+    '''load data from the speclite package'''
+
+    # result is spectrum or list of spectra
+    spectra = None
+
+    # load each band sperately
+    for band in bands:
+
+        # get resource filename from name and bands
+        filter_name = f'{name}-{band}'
 
-    # download DECam filter data
-    filename = download_file(
-            'http://www.ctio.noao.edu/noao/sites/default/files/DECam/STD_BANDPASSES_DR1.fits')
+        # load the filter response as a speclite.FilterResponse object
+        filter_object = speclite.filters.load_filter(filter_name)
 
-    # load the data file
-    data = astropy.table.Table.read(filename, format='fits')
+        # get the spectral axis as astropy quantity
+        spectral_axis = units.Quantity(filter_object._wavelength, units.angstrom)
 
-    # set units
-    data['LAMBDA'].unit = units.angstrom
+        # get throughput
+        throughput = filter_object.response * units.dimensionless_unscaled
 
-    # get the spectral axis
-    spectral_axis = data['LAMBDA'].quantity
+        # construct the Spectrum1D
+        spectrum = specutils.Spectrum1D(spectral_axis=spectral_axis, flux=throughput)
 
-    # load requested bands
-    throughput = []
-    for band in bands:
-        throughput.append(data[band])
-    throughput = np.squeeze(throughput)*units.dimensionless_unscaled
+        # combine with existing
+        spectra = combine_spectra(spectra, spectrum)
 
-    # return the bandpasses as Spectrum1D
-    return specutils.Spectrum1D(spectral_axis=spectral_axis, flux=throughput)
+    return spectra
 
 
 spectrum_loaders = [
     # bandpasses
-    ('(Johnson)_(U)?(B)?(V)?', skypy_data_loader, 'bandpasses'),
-    ('(Cousins)_(R)?(I)?', skypy_data_loader, 'bandpasses'),
-    ('DECam_(g)?(r)?(i)?(z)?(Y)?', decam_loader),
+    ('(decam2014)_(u)?(g)?(r)?(i)?(z)?(Y)?', speclite_loader),
+    ('(sdss2010)_(u)?(g)?(r)?(i)?(z)?', speclite_loader),
+    ('(wise2010)_(W1)?(W2)?(W3)?(W4)?', speclite_loader),
+    ('(hsc2017)_(g)?(r)?(i)?(z)?(y)?', speclite_loader),
+    ('(lsst2016)_(u)?(g)?(r)?(i)?(z)?(y)?', speclite_loader),
+    ('(bessell)_(U)?(B)?(V)?(R)?(I)?', speclite_loader),
+    ('(BASS)_(g)?(r)?', speclite_loader),
+    ('(MzLS)_(z)?', speclite_loader),
 
```

Comment:
We don't want to mirror the list of speclite filters here, so we need to think about changing this in a second step.

Reply:
Keep them for now. They are still needed because this PR still uses `load_spectral_data`. If we refactor and deprecate that function in a later PR we can cut this list then.

Reply:
I am also not really happy with it. I just did it to have the first implementation. On the other hand, I thought that it might be not that bad as it is very comprehensible. 

Do you have any suggestions for different names?